{{ cookiecutter.repo_name }}/{{ cookiecutter.project_name }}/{{ cookiecutter.plugin_name.lower().replace(' ', '') }}plugin.py

```diff
@@ -83,8 +99,11 @@ def register_plugin(self):
 
     def on_first_registration(self):
         """Action to be performed on first plugin registration."""
-        self.main.tabify_plugins(self.main.help, self)
+        # Define where the plugin is going to be tabified next to
+        # As default, it will be tabbed next to the ipython console
+        self.tabify(self.main.help)
```

Comment:
This is different for Spyder 3 and 4. For Spyder 3 it needs to be

    self.main.tabify_plugins(self.main.help, self)

and for Spyder 4 what you already have in this line.